### PR TITLE
coprosssor: do not share backoffer by tasks.

### PR DESCRIPTION
**Pull request by disksing**
@shenli @siddontang

### CODE REVIEW

File: store/tikv/coprocessor.go [hunk 1]
```diff
@@ -302,8 +302,8 @@ type copIterator struct {
 
 // Pick the next new copTask and send request to tikv-server.
 func (it *copIterator) work() {
-	bo := NewBackoffer(copNextMaxBackoff)
 	for {
+		bo := NewBackoffer(copNextMaxBackoff)
```
Comment (shenli) [inline store/tikv/coprocessor.go]: It is better to move it to line 325.

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@coocood PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**